.gitignore

```diff
@@ -2,5 +2,7 @@
 /vendor/
 /composer.lock
 /.phpsu/
+/tests/.phpsu
```

Comment:
This should never happen :/
- We need to fix this before it can lay eggs

Reply:
Open an issue then as this has nothing to to with this branch and this branch already contains more than it was supposed to have